content/homebrew-build-provenance.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Homebrew's Sigstore-powered provenance is in beta"
+date = "2024-05-16"
+tags = ["sigstore", "security", "homebrew"]
+draft = false
+author = "William Woodruff (Trail of Bits), Hayden Blauzvern (Google)"
+type = "post"
++++
+
+Last November, [Alpha-Omega] and [Trail of Bits] announced a
+[collaboration to bring build provenance to homebrew-core].
+
+Today, we are pleased to announce that the core of that work is live
+and in public beta: [homebrew-core] is now using Sigstore to cryptographically
+attest to all bottles built in the official [Homebrew] CI. This follows
+last year's [npm provenance] feature, making Homebrew the second major
+packaging ecosystem to adopt Sigstore!
```

Comment:
Can we have a line about continuing Sigstore's mission (linking to https://github.com/sigstore/community/blob/main/ROADMAP.md#mission-aka-our-purpose) to support frictionless & transparent provenance on all artifact registries?